src/types/BeachballOptions.ts

```diff
@@ -42,6 +42,7 @@ export interface CliOptions {
   prereleasePrefix?: string | null;
   configPath?: string;
   commit?: boolean;
+  multipleChangesPerChangefile?: boolean;
```

Comment:
This name is a bit verbose, especially if it has to be specified on the command line. I wonder if it would be possible to come up with something more concise but still clear? 

Whatever the name is, it should probably be added to `RepoOptions` in addition or instead (IMO this would make more sense to enable in a config file than on the command line), and documented in `docs/overview/configuration.md`.

Reply:
I agree with the comment here about putting in "RepoOptions" and generally be checked into a beachball.config.js rather than in a CLI...

Reply:
Sounds great. Thanks.